packages/web/src/components/apps/workspace/preview.tsx

```diff
@@ -41,11 +41,6 @@ export function Preview(props: PropsType) {
 
       return (
         <div className={cn('w-full h-full', props.className)}>
-          <div className="absolute">
-            <button onClick={() => addError({ type: 'vite_error', contents: 'Bogus error' })}>
-              Trigger
-            </button>
-          </div>
           <iframe className="w-full h-full" src={url} title="App preview" />
         </div>
```

Comment:
😬 I forgot I left this in when I merged my [last change](https://github.com/srcbookdev/srcbook/pull/365)... sorry!

Reply:
We're moving fast and I love it. Tiny price to pay